CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Unify breadcrumbs behaviour with other Grafana Apps and main core ([#1906](https://github.com/grafana/oncall/issues/1906))
+- Added index on multiple columns in `user_management_user` table. This substantially speeds up queries used for showing the schedules of users.
```

Comment:
Would you update your changelog entry to be in one line?

Reply:
I will when I get a chance.

Reply:
The linter in the github actions wants this to be 2 lines